packages/calling/src/CallingClient/CallingClient.ts

```diff
@@ -504,6 +435,40 @@ export class CallingClient extends Eventing<CallingClientEventTypes> implements
   public getLines(): Record<string, ILine> {
     return this.lineDict;
   }
+
+  /**
+   *
+   */
+  public getActiveCalls(): Record<string, ICall[]> {
```

Comment:
needs some description for the function

Reply:
Added.

Reply:
@Kesari3008  Can we have an optional parameter `lineId` which will only return calls from a a single line?

```suggestion
  public getActiveCalls(lineId?: string): Record<string, ICall[]> {
```

Reply:
Yeah, we can do that I guess. Let's discuss on it Monday

Reply:
We will do this as part of separate PR and discuss the approach for the same